core/views.py

```diff
@@ -560,7 +560,11 @@ class BaseRedirectView(View):
     @staticmethod
     def get_latest_library_version():
         """Return the latest version for a given library."""
-        return Version.objects.most_recent().stripped_boost_url_slug
+        most_recent_version = Version.objects.most_recent()
+        if most_recent_version is None:
+            # No versions imported - new dev setup or test
+            return "1.87.0"
```

Comment:
Is there another way to do this? This is bound to be forgotten after a future boost release and be opaque about why 1.87.0 is selected. I'd almost prefer if it just logged an error stating the user needs to update their libraries with an exception given it's just going to be for dev/test use.